vispy/app/backends/_tk.py

```diff
@@ -0,0 +1,781 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+
+"""
+vispy backend for Tkinter.
+"""
+
+from __future__ import division
+
+from time import sleep
+
+from ..base import (BaseApplicationBackend, BaseCanvasBackend,
+                    BaseTimerBackend)
+from ...util import keys
+from ...util.ptime import time
+
+
+# -------------------------------------------------------------------- init ---
+
+# Import
+try:
+    import sys
+    import os
+
+    _tk_on_linux = sys.platform.startswith('linux')
+    _tk_on_darwin = sys.platform == 'darwin'
+    _tk_on_windows = sys.platform.startswith('win')
+
+    try:
+        import tkinter as tk  # Python >= 3
+    except ModuleNotFoundError:
+        import Tkinter as tk  # Python < 3
+    from OpenGL import GL
+    import pyopengltk
+    OpenGLFrame = pyopengltk.OpenGLFrame
+except (ModuleNotFoundError, ImportError):
+    available, testable, why_not, which = \
+        False, False, "Could not import Tkinter or pyopengltk, module(s) not found.", None
+
+    class OpenGLFrame(object):
+        pass
+else:
+    available, testable, why_not = True, True, None
+    which_pyopengltk = getattr(pyopengltk, "__version__", "???")
+    which = "Tkinter " + str(tk.TkVersion) + " (with pyopengltk " + which_pyopengltk + ")"
+
+
+def _fix_tcl_lib():
+    """
+    It is possible that the Tcl library cannot be found when the Python environment
+    does not have the proper variables, so we force them here (mostly when running tests).
+    From: https://github.com/enthought/Python-2.7.3/blob/master/Lib/lib-tk/FixTk.py
```

Comment:
Is this also true for Python 3? Or is this just a Python 2 specific problem?

Reply:
I have had this problem in the past with Python 2, in particular in conjunction with one-file PyInstaller applications. So it might still occur with Python3, albeit it being rare.

Reply:
According to https://github.com/python-pillow/Pillow/pull/2455 this is only for Windows and only with Python 2.7. I think it can be removed.